GiniCapture/Classes/Core/Screens/Help/HelpMenuViewController.swift

```diff
@@ -10,90 +10,63 @@ import UIKit
         case noResultsTips
-        case openWithTutorial
-        case supportedFormats
-        
-        var title: String {
-            switch self {
-            case .noResultsTips:
-                return .localized(resource: HelpStrings.menuFirstItemText)
-            case .openWithTutorial:
-                return .localized(resource: HelpStrings.menuSecondItemText)
-            case .supportedFormats:
-                return .localized(resource: HelpStrings.menuThirdItemText)
-            }
-        }
-        
-        var viewController: UIViewController {
-            let viewController: UIViewController
-            switch self {
-            case .noResultsTips:
-                let title: String = .localized(resource: ImageAnalysisNoResultsStrings.titleText)
-                let topViewText: String = .localized(resource: ImageAnalysisNoResultsStrings.warningHelpMenuText)
-                viewController = ImageAnalysisNoResultsViewController(title: title,
-                                                                      subHeaderText: nil,
-                                                                      topViewText: topViewText,
-                                                                      topViewIcon: nil)
-            case .openWithTutorial:
-                viewController = OpenWithTutorialViewController()
-            case .supportedFormats:
-                viewController = SupportedFormatsViewController()
-            }
-            
-            return viewController
-            
-        }
+        case helpItem
     }
-    
-    lazy var items: [Item] = {
-        var items: [Item] = [ .noResultsTips]
-        
-        if giniConfiguration.shouldShowSupportedFormatsScreen {
-            items.append(.supportedFormats)
-        }
+
+    /**
+     The `MenuItem` class represents a menu item with its detail view
+     
+     - Parameters:
+        - title: A title for the help menu item. We recommend to define it as a localized string if you support multiple languages
+        - viewController: A custom view controller which represents a detail view for help menu item
+        - presentationMode: An optional parameter, will be set as `.helpItem` as a default, responsible for navigation bank from the detailed view of the  help menu item
+     
+     */
+    @objc public class MenuItem: NSObject {
+        public var title: String
+        public var viewController: UIViewController
+        public var presentationMode: ItemPresentationMode?
         
-        if giniConfiguration.openWithEnabled {
-            items.append(.openWithTutorial)
+        public init (title: String, viewController: UIViewController, presentationMode: ItemPresentationMode? = .helpItem) {
+            self.title = title
+            self.viewController = viewController
+            self.presentationMode = presentationMode
         }
-        
-        return items
-    }()
+    }
     
+    lazy var menuItems: [MenuItem] = []
```

Comment:
We change the type to `Item`:
```suggestion
    lazy var menuItems: [Item] = []
```

Continued below ->